uptane-standard.md

```diff
@@ -714,7 +720,7 @@ There may be several different filenames that all refer to the same image binary
 
 #### Send latest time to secondaries {#send_time_primary}
 
-The primary SHOULD send the time to each ECU. The secondary will verify the time message, then overwrite its current time with the received time. The primary MAY omit this step if the secondary has its own way of loading and verifying the time.
+The primary SHALL send the time to each ECU. The secondary will verify the time message, then overwrite its current time with the received time. The primary MAY omit this step if the secondary has its own way of loading and verifying the time, or cannot verify the time has detailed here.
```

Comment:
```suggestion
The primary SHALL send the time to each ECU. The secondary will verify the time message, then overwrite its current time with the received time. The primary MAY omit this step if the secondary has its own way of loading and verifying the time, or cannot verify the time as detailed here.
```

Reply:
I don't understand why we would use `SHALL` if we the immediately say "actually you can skip this" immediately afterwards. Sounds like a `SHOULD` to me.

Reply:
Thanks for the comment, Patrick.

My thought is: if you make this a 'SHOULD' then this whole paragraph becomes conditional.
Therefore, there is *no* requirement for the primary to send the time to each ECU.
I believe this is more problematic than the current state.

Reply:
Two things:

1. The caveat that the primary MAY omit the step if the secondary isn't capable of verifying the time is a catch-all. It is, most definitely, a SHOULD.
2. It would perhaps be slightly more proper to use the RFC2119 keyword CONDITIONALLY REQUIRED. Something like "Unless the secondary ECU has its own way of verifying the time, or does not have the capacity to verify a time message, the primary is CONDITIONALLY REQUIRED to send the time to each ECU. The secondary will verify the time message, then overwrite its current time with the received time."

Reply:
Updated per your recommendation.